.github/integration/scripts/make_sda_credentials.sh

```diff
@@ -82,4 +82,9 @@ if [ ! -f "/shared/keys/ssh" ]; then
     }
 ]
 EOD
+fi
+
+if [ ! -f "/shared/grpcurl" ]; then
+    echo "downloading grpcurl"
+    curl -s -L https://github.com/fullstorydev/grpcurl/releases/download/v1.8.9/grpcurl_1.8.9_linux_x86_64.tar.gz | tar -xz -C /shared/ && chmod +x /shared/grpcurl
```

Comment:
i would view `go install github.com/fullstorydev/grpcurl/cmd/grpcurl@latest` but not sure we have go where we run the scripts, as it will be difficult to update the version. similar thing with the outdated crypt4gh version for the keys ... maybe smth like

```suggestion
    latest_tag=$(curl  "https://api.github.com/repos/fullstorydev/grpcurl/tags" | jq -r '.[0].name')
    curl -s -L "https://github.com/fullstorydev/grpcurl/releases/download/${latest_tag}/grpcurl_${latest_tag:1}_linux_x86_64.tar.gz" | tar -xz -C /shared/ && chmod +x /shared/grpcurl
```